lib/cell-manager.coffee

```diff
@@ -0,0 +1,51 @@
+_ = require 'lodash'
+
+module.exports = CellManager =
+    removeAllBreakPoints: ->
```

Comment:
I'm fine with it because it has a close parallel to the notebook cells. No real suggestions from me here.